scripts/core/exp-command-buffer.yml

```diff
@@ -588,6 +588,75 @@ returns:
     - $X_RESULT_ERROR_OUT_OF_RESOURCES
 --- #--------------------------------------------------------------------------
 type: function
+desc: "Append a USM Prefetch command to a command-buffer object"
```

Comment:
We should say under `details:` that this is a backend defined hint that isn't required for correctness.